tests/test_tutorial/test_custom_response/test_tutorial001.py

```diff
@@ -0,0 +1,36 @@
+from fastapi.testclient import TestClient
```

Comment:
Why was this file added in this pr?

Reply:
Since this PR also re-adds `UJSONResponse` that was removed in starlette 0.14.1 and the response code should be tested as well. So I followed suite by adding a test case similar to the ORJSONResponse.